source/content/plugins-known-issues.md

```diff
@@ -976,11 +977,21 @@ You will need to make this change every time the plugin is updated.
 
 ___
 
+## WP Reset
+
+<ReviewDate date="2021-11-04" />
+
+**Issue 1:** Some features of the [WP Reset](https://wordpress.org/plugins/wp-reset/) plugin can not be used on the Pantheon platform. Features such a file reset and restore do not work because Staging and Production environments are immutable and backups help with restore and data rollbacks. 
```

Comment:
how about adding in the tone of they're "not necessary" to highlight the benefit of our platform's design.

for example, "environments are immutable making WP Reset not necessary, and the platform's built-in backups are already in place to help with restore and data rollbacks." yadda yadda